.chromer/cli/bundle.js

```diff
@@ -1,18 +1,42 @@
-const { readFile, writeFile } = require('fs-extra');
 const { resolve } = require('path');
+const { readFile, writeFile, pathExists, readJson } = require('fs-extra');
 const crx = require('crx');
+const chalk = require('chalk')
+const { paramCase } = require('change-case')
 
 
 // TODO: Support .crxignore
 const main = async () => {
   try {
+    const distDir = resolve(process.cwd(), 'dist')
+    const builtManifestPath = resolve(distDir, 'manifest.json')
+    const keyPath = resolve(process.cwd(), 'keys/key.pem')
+
+    const isManifestExists = await pathExists(builtManifestPath)
+    const isKeyExists = await pathExists(keyPath)
+
+    if (!isManifestExists) {
+      console.error(`${chalk.bold('manifest.json')} does not exist. Run ${chalk.dim('yarn build')} before.`)
```

Comment:
Perhaps move all these guard-alike procedures to the helper.js?

Reply:
Don't really like 'helper' name, but I will doing something similar

Reply:
I'm not sure that we really need to extract this method to separate module.
It's really non reusable methods with `bundle` context.  Also there is collision with `src/manifest.json` and `dist/manifest.json`. So it's will really strange if we are will move all file existing checks in the one file

Reply:
Would you perhaps get rid of helper.js?

Reply:
Will do, but not in this PR. I find useful functions like `assertNoFile` (rewrited `assertNoManifest`), but not for this PR ;)